tests/v2/validation/rbac/rbac.go

```diff
@@ -46,7 +48,7 @@ const (
 )
 
 type ClusterConfig struct {
-	nodeRoles            []string
+	nodesAndRoles        []nodepools.NodeRoles `json:"nodesAndRolesRKE1" yaml:"nodesAndRolesRKE1" default:"[]"`
```

Comment:
actually one last thing do we need the json tags here?

Reply:
I guess not 🥲